components/FilterBar.js

```diff
@@ -1,44 +1,68 @@
 import { useState } from "react";
 import useFilters from "@/hooks/useFilters";
-
-export default function FilterBar({
-  onChange,
-  groupedCityEvents,
-  groupedCategoryEvents,
-  filters,
-  reset,
-}) {
+import styled from "styled-components";
+import Button from "./Button";
+import useSWR from "swr";
+const StyledDiv = styled.div`
```

Comment:
Let's call it `StyledContainer` at least, to give a bit more of indication to the developer ^^